misc/scripts/search.sh

```diff
@@ -67,7 +67,7 @@ if [[ $PACKAGE == *@* ]]; then
 	
 	fancy_message warn "$IRed$REPONAME$NC is not on your repo list or does not exist"
 	error_log 3 "search $PACKAGE@$REPONAME"
-	return 1	
+	exit 1
```

Comment:
Same thing here

Reply:
There is no function this is in, so having return is wrong.